.eslintrc.js

```diff
@@ -1,17 +1,12 @@
 module.exports = {
   root: true,
-  parserOptions: {
-    ecmaVersion: 6,
-    sourceType: 'script'
-  },
-  extends: [
-    'eslint:recommended'
-  ],
   env: {
-    'node': true,
     'es6': true
   },
+  extends: [
+    'plugin:turbopatent/node'
+  ],
   rules: {
-    "no-unused-vars": [ "error", { vars: "all", args: "none" }]
+    'no-var': 'error'
```

Comment:
I don't think you need this since we set it [here](https://github.com/PatentNavigation/eslint-plugin-turbopatent/blob/master/config/base.js#L47)